src/dynamoTable.py

```diff
@@ -0,0 +1,27 @@
+import boto3
+from boto3.dynamodb.conditions import Key, Attr
+
+class dynamoTable(object):
+    
+    """Initialize table
+       Can be buildings, machines, users
+    """
+    def __init__(self, tableType):
+        allowedTableTypes = ['Buildings', 'Machines', 'Users']
+        if tableType not in allowedTableTypes:
+            raise ValueError('Invalid table type! Must be one of: {}'.format(allowedTableTypes))
+        dynamodb = boto3.resource('dynamodb', region_name = 'us-east-2')

```

Comment:
You should actually be able to omit the `'us-east-2'` and the Lambda will automatically figure it out based on the region it's in. Will make things easier if someone decides to deploy in another region.